Core/ShaderAnalysis/src/shae.py

```diff
@@ -324,6 +326,10 @@ def GetBinEncodingText(self, comment):
         foundBinCode = re.search(r' *[0-9a-fA-F]+: *([0-9a-fA-F]{8})( *[0-9a-fA-F]{8})?', comment)
         if foundBinCode:
             result = comment[comment.find(':')+1 :].lstrip(' ')
+        # Attempt to match comments containing only the bin code.
+        foundBinCode = re.search(r' *([0-9a-fA-F]{8})( *[0-9a-fA-F]{8})?', comment)
+        if foundBinCode:
+            result = comment.lstrip(' ')
```

Comment:
We sometimes see a runtime error (ValueError: invalid literal for int() with base 16: '000000000054:') later in the script (line 119) with this change.  If we make line 332 identical to line 328, we never see the syntax error.

Reply:
I believe the 'else' I added in response to your other comment should fix this (since now the logic should be unchanged except if the input looks like what I'm trying to support)